frontend/src/settings.tsx

```diff
@@ -120,11 +193,37 @@ const Settings: React.FC = () => {
         <TextInput
           type="text"
           {...settingsForm.getInputProps("customOrder")}
-          label="Custom letter order. Leave blank to use alphabetical. All letters not specified will be in alphabetical order."
+          label="Custom letter order"
+          description="Leave blank to use alphabetical. All letters not specified will be in alphabetical order."
           placeholder="AEIOU..."
           mt="lg"
-          size="lg"
         />
+
+        <Divider mt="lg" />
+
+        <Text size="xl" mt="lg">
+          Scheduling Settings
+        </Text>
+
+        <NumberInput
+          label="Desired retention rate"
+          description="The percentage of words you want to remember. Choose from 70% to 97%."
```

Comment:
I think the wording here could be something like:

```
Target probability of recalling a word during review. The algorithm adjusts scheduling to maintain this rate. Choose 70%–97% (default: 90%).
```

what do you think? "% of words you want to remember" isn't quite accurate.

Reply:
This phrasing sounds good to me! will clean up the wording (and implementation) this evening